pkgs/applications/networking/browsers/firefox/update.nix

```diff
@@ -15,7 +16,12 @@
 
 writeScript "update-${attrPath}" ''
   #!${runtimeShell}
-  PATH=${lib.makeBinPath [ common-updater-scripts coreutils curl gnugrep gnused xidel ]}
+  PATH=${lib.makeBinPath [ common-updater-scripts coreutils curl gnugrep gnupg gnused xidel ]}
+
+  set -eux
+  HOME=`mktemp -d`
+  export GNUPGHOME=`mktemp -d`
+  gpg --import ${../firefox-bin/mozilla.asc}
```

Comment:
Can we just fetch the key from the web and add a hash here?

Reply:
Changed to use `gpg --receive-keys`.